### PR TITLE
Update to RepoToolset beta2-62730-01 and sync bootstrap scripts

### PR DESCRIPTION
**Pull request by tmat**
I have restructured build.sh and build.ps1 to match each other closer.

### CODE REVIEW

File: eng/common/build.ps1 [hunk 1]
```diff
@@ -56,113 +55,168 @@ if ($help -or (($properties -ne $null) -and ($properties.Contains("/help") -or $
...
-  } else {
-    $logCmd = ""
+  return $vsInstallDir
+}
+
+function InitializeToolset {
+  if (!(Test-Path $ToolsetBuildProj)) {
+   $proj = Join-Path $TempDir "_restore.proj"   
+   '<Project Sdk="RoslynTools.RepoToolset"><Target Name="NoOp"/></Project>' | Set-Content $proj
+    & $BuildDriver $BuildArgs $proj /t:NoOp /m /nologo /clp:None /warnaserror /v:$verbosity /p:NuGetPackageRoot=$NuGetPackageRoot /p:__ExcludeSdkImports=true
```
Comment (chcosta) [inline eng/common/build.ps1]: Consider adding some of the redundant args between this function and "Build" into "BuildArgs".  ie `/m /nologo /warnaserror /v:$verbosity /p:NuGetPackageRoot=$NuGetPackageRoot`

File: eng/common/build.sh [hunk 2]
```diff
@@ -56,36 +57,31 @@ while (($# > 0)); do
...
-      echo "  --pack           Package build outputs into NuGet packages and Willow components"
+      echo "  --restore                Restore dependencies"
+      echo "  --build                  Build solution"
+      echo "  --rebuild                Rebuild solution"
+      echo "  --test                   Run all unit tests in the solution"
+      echo "  --sign                   Sign build outputs"
+      echo "  --pack                   Package build outputs into NuGet packages and Willow components"
       echo ""
       echo "Advanced settings:"
       echo "  --projects <value>     Semi-colon delimited list of sln/proj's to build. Globbing is supported (*.sln)"
```
Comment (chcosta) [inline eng/common/build.sh]: Inconsistent alignment before "  Semi-colon delimited..."

File: eng/common/build.sh [hunk 3]
```diff
@@ -137,189 +133,179 @@ function ReadJson {
...
+  if [[ -z $projects ]]; then
+    projects="$repo_root/*.sln"
   fi
-  
+
   if [[ -z $NUGET_PACKAGES ]]; then
-    if [[ "$officialbuild" == true ]]; then
-      export NUGET_PACKAGES="$reporoot/packages"
+    if [[ "$official_build" == true ]]; then
+      export NUGET_PACKAGES="$repo_root/packages"
```
Comment (chcosta) [inline eng/common/build.sh]: Did you intend to remove this code path with `"$repo_root/packages"` and also remove the `MakeGlobalSdkAvailableLocal` code?

Comment (tmat) [reply eng/common/build.sh]: Yes, I actually have a newer version. Will update later this week.

**Pull request closed without merging**